slackdump/slackdump.py

```diff
@@ -0,0 +1,62 @@
+import time
+import hashlib
+
+from slackclient import SlackClient
+from pymongo import MongoClient
+
```

Comment:
@UmangThapliyal @ManishaBayya  There is no need for the extra line because these all are external modules only and remove `.` before the `.slackhistory`.

Reply:
For python3 we have to use . to import

Reply:
@ManishaBayya Is there a particular reason, to use `.` in python3?

Reply:
@chatrapathik  slackclient and pymongo are 3rd party modules and slackhistory is our module. So we separated them.